pennylane/templates/state_preparations/mottonen.py

```diff
@@ -208,13 +224,17 @@ def MottonenStatePreparation(state_vector, wires):
     The state is prepared via a sequence
     of "uniformly controlled rotations". A uniformly controlled rotation on a target qubit is
     composed from all possible controlled rotations on said qubit and can be used to address individual
-    elements of the state vector. In the work of Mottonen et al., the inverse of their state preparation
+    elements of the state vector. In the work of Möttönen et al., the inverse of their state preparation
     is constructed by first equalizing the phases of the state vector via uniformly controlled Z rotations
     and then rotating the now real state vector into the direction of the state :math:`|0\rangle` via
     uniformly controlled Y rotations.
 
     This code is adapted from code written by Carsten Blank for PennyLane-Qiskit.
 
+    .. note::
+
+        The final state is only equal to the input state vector up to a global phase.
```

Comment:
This is good to keep in mind when we add global phases to the library eventually. Do we know where in the code would have to be adjusted to get correct global phase?

Reply:
No, I think that the original algorithm does not cater for this. This may be a tough piece of theory work.